src/issues/components/IssueVote.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <QCard
+    class="wrapper"
+    :class="{showOverlay}">
+    <QBtn
+      v-if="showOverlay"
+      class="absolute-center"
+      style="z-index: 1"
+      color="primary"
+      @click="setToZero()"
+      v-t="'ISSUE.VOTING.BTN_START'"
+    />
+    <div>
+      <QCardTitle>
+        {{ $t('CONFLICT.VOTING.HEADLINE', { userName: issue.affectedUser.displayName }) }}
+      </QCardTitle>
+      <QCardMain>
+        {{ $t('ISSUE.VOTING.DAYS_LEFT', { count: days }) }}
+        <QProgress
+          :percentage="progress"
+          style="height: 8px"
+          color="secondary"
+        />
+      </QCardMain>
+    </div>
+    <div
+      class="content"
+    >
+      <QBtn
+        class="absolute-right"
+        v-if="!showOverlay"
+        round
+        flat
+        color="red"
+        @click="setToNull()"
+      >
+        <QIcon name="fas fa-times" />
+        <QTooltip
+          v-t="'ISSUE.VOTING.BTN_DELETE'"
+        />
+      </QBtn>
+      <QCardMain
+        v-for="o in edit"
+        :key="o.id"
+      >
+        {{ getTitle(o.type) }}
+        <QSlider
+          v-model="o.yourScore"
+          :label-value="getLabel(o.yourScore)"
+          :min="-2"
+          :max="2"
+          :step="1"
+          label-always
+          snap
+          markers
+        />
+      </QCardMain>
+      <QBtn
+        type="submit"
+        color="secondary"
+        v-t="'BUTTON.SUBMIT'"
+        @click="$emit('saveScores', results)"
+      />
+    </div>
+  </QCard>
+</template>
+
+<script>
+import {
+  QCard,
+  QCardTitle,
+  QCardMain,
+  QSlider,
+  QProgress,
+  QBtn,
+  QIcon,
+  QTooltip,
+} from 'quasar'
+
+import distanceInWordsStrict from 'date-fns/distance_in_words_strict'
+import differenceInHours from 'date-fns/difference_in_hours'
+import cloneDeep from 'clone-deep'
+// import { objectDiff } from '@/utils/utils'
+
+export default {
+  components: {
+    QCard,
+    QCardTitle,
+    QCardMain,
+    QSlider,
+    QProgress,
+    QBtn,
+    QIcon,
+    QTooltip,
+  },
+  props: {
+    issue: {
+      type: Object,
+      required: true,
+    },
+  },
+  data () {
+    return {
+      edit: null,
+    }
+  },
+  watch: {
+    options: {
+      immediate: true,
+      handler (current, previous) {
+        /*
```

Comment:
I _think_ it should be fine without the commented code. 
It means that the users' pending changes will get overwritten when a change comes in (e.g. a websocket message), so we need to check that case.